tool/monitor_directory.py

```diff
@@ -0,0 +1,52 @@
+import argparse
+import os
+import time
+import datetime
+
+from firebase_util import init_firebase_client
+from katikati_pylib.logging import logging
+
+CRYPTO_TOKEN_PATH = None
+DEFAULT_INTERVAL = 600 # 10 minutes
+firebase_client = None
+log = None
+
+def get_size_and_upload(dirpath, project):
+    global log
+    if log is None:
```

Comment:
I think it makes more sense now to create the logger under main rather than here since this will be run as a standalone script rather than being imported somewhere else. There will also be no need then to keep a global reference to `CRYPTO_TOKEN_PATH`